src/components/Visualizations/Containers/AuthorNetworkPageContainer.tsx

```diff
@@ -0,0 +1,393 @@
+import { IADSApiSearchParams, useSearch, useVaultBigQuerySearch } from '@api';
```

Comment:
I think we should probably break this file apart into several components.  They could all live inside this file, but I think that would make it simpler.

Reply:
@thostetler I've moved the new changes for D3 and the refactoring to #233 
This PR has nothing new since you last reviewed.
Let me know if you feel this PR can be merged